lighthouse-core/scripts/generate-timing-trace.js

```diff
@@ -50,7 +50,7 @@ function saveTraceFromCLI() {
   // eslint-disable-next-line no-console
   console.log(`
   > Timing trace file saved to: ${traceFilePath}
-  > Open this file in chrome://tracing
+  > Open this file in DevTools perf panel (or if it's a -G/A run, use chrome://tracing)
```

Comment:
does `-G/A run` refer to a "`-G` or an `-A` run", or a `-GA` run or a regular run (that did both gathering and auditing)?